theClanless/src/main/java/theClanless/powers/PrecisionPower.java

```diff
@@ -0,0 +1,57 @@
+package theClanless.powers;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import theClanless.cards.AbstractClanlessCard;
+import theClanless.theClanlessMod;
+import theClanless.util.TextureLoader;
+
+import static theClanless.theClanlessMod.makePowerPath;
+
+public class PrecisionPower extends AbstractPower {
```

Comment:
I WAS going to make precision grab a card from your draw pile and reduce it's cost. But this works too.  ;)

Reply:
Your idea is good. I can remove this card. Just say the word.